tests/e2e/crud/crud_test.go

```diff
@@ -104,14 +106,12 @@ func init() {
 	flag.Parse()
 
 	var err error
+	kubeClient, err = client.New(*kubeConfig)
```

Comment:
In this case, is it necessary that the environment in which the test is run always have information about the connection to the Kubernetes cluster? 🤔

Reply:
Yes.

`TestE2EReadReplica` needs to be dependent on k8s client anyway so I guess I should revert this and check this in `TestE2EReadReplica`? That way other e2e tests can run without k8s clinet.

Reply:
I fixed that way [ed4e8b3](https://github.com/vdaas/vald/pull/2298/commits/ed4e8b3ec09da7f635624ec3110626b39a676e49) since I think it's better anyway.

Let me know if this is not what you intended.

Reply:
> so I guess I should revert this and check this in TestE2EReadReplica?

Thank you for reply :pray:
Yes, I think that would be better!!

Reply:
I reviewed your changes! It looks good to me! 🙏 
Thank you 🙏